### DISCUSSION TITLE
Audio drops and UI issues Mac

### DISCUSSION THREAD

**Discussion by rvaix** [apohl79/audiogridder#400] [Help]
Hello all, 

I use AG plug-in 1.1 on Cubase 10.5 on MBP with 10.15 and serveur on a MBA 10.14.
Daw's buffer size is at 1024, server buffer size is at maximum. 
Both machines connect fairly well. Except that I get audio drops (CPU surcharge? ) only by inserting the G plug-in, even without any instance. 
When I remote an FX, the UI sometimes appears then disappears, sometimes it does not appear at all. 
I've tried with AU + VST internal plug-ins and Powercore ones. It's even worse with the Powercore. The generic interface doesn't  show up with the parameters name. 
Any suggestions to improve the set up or this technology is simply not yet reliable ? 
Thanks for your comments. 

Rv

**Comment by apohl79** [comment]
When you say server buffer is at max, do you mean you increased the AG buffer in the AG plugin?

If you still get audio drops, your network is likely the problem.

What latency do you see in the plugin monitor and in the statistics window of the plugin and the server?

**Reply by rvaix** [reply]
Yes, I put the buffer size to its max. 30 blocks (+ 696 ms) from the plug in. 
And the various latencies.
<img width="403" alt="Capture d’écran 2021-02-21 à 23 00 39" src="https://user-images.githubusercontent.com/79191418/108640374-8b405700-7499-11eb-901f-e57accf5bc5f.png">
<img width="443" alt="Capture d’écran 2021-02-21 à 23 06 05" src="https://user-images.githubusercontent.com/79191418/108640380-94312880-7499-11eb-8814-e111bd50b720.png">
![Capture d’écran 2021-02-21 à 23 01 47](https://user-images.githubusercontent.com/79191418/108640833-23d7d680-749c-11eb-8687-cb65ffc5b9db.png)

Thanks

**Reply by apohl79** [reply]
With these latency numbers you should be able to use 0 or 1 block buffering in AG. Do you have issues with other plugins too? Can you share a plugin and server logfile?

**Comment by CaBuffalo** [comment]
Is it possible for you to make a short video showing your problem?

On Sun, Feb 21, 2021 at 10:25 AM Andreas Pohl <notifications@github.com>
wrote:

> When you say server buffer is at max, do you mean you increased the AG
> buffer in the AG plugin?
>
> If you still get audio drops, your network is likely the problem.
>
> What latency do you see in the plugin monitor and in the statistics window
> of the plugin and the server?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/400#discussioncomment-391287>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6EBRYEPW2TXQ2CQZYLTAFFZNANCNFSM4X7ABW6Q>
> .
>


-- 
Kevin Ma
Supported by Gmail.
